test/load.cpp

```diff
@@ -298,13 +306,13 @@ int main(int argc, char *argv[]){
   }
   {
     // Write raw JSON
-    std::string fName = outFile.str()+".json";
+    std::string fName = config.getString("outputdir")+outFile.str()+".json";
```

Comment:
This will result in `-o /tmp` to write to `/tmpFILE.json`, no?

Reply:
Oops, I fixed this earlier but forgot to push the latest commit. Fixed.